Volume/Views/MainView/HomeList.swift

```diff
@@ -137,6 +137,7 @@ struct HomeList: View {
                                 .font(.begumRegular(size: 18))
                                 .foregroundColor(.volume.orange)
                                 .padding([.leading])
```

Comment:
I know you didn't write this, but could you change this to
`.padding(.leading)`